# This is a message that holds data to describe the position and angle wrist of excavaROB.
#
# The state of wrist of excavaROB is defined by:
#  * the wrist position (m),
#  * the wrist angle (rad), 
#  * the wrist velocity (m/s) and
#  * the wrist angular velocity (rad/s)



Header header

WristPoint wrist_pos
WristAngle wrist_angle_pos

WristPoint wrist_vel
WristAngle wrist_angle_vel

WristPoint wrist_acc
WristAngle wrist_angle_acc
